common/app/views/fragments/head.scala.html

```diff
@@ -1,16 +1,13 @@
-@(metaData: model.MetaData, projectName: Option[String] = None, head: Html)(implicit request: RequestHeader)
-@import conf.Static
-@import conf.Configuration
-@import views.support.OmnitureAnalyticsData
+@(page: model.Page, projectName: Option[String] = None, head: Html)(implicit request: RequestHeader)
 
 <meta charset="utf-8" />
-<title>@views.support.Title(metaData)</title>
+<title>@views.support.Title(page)</title>
 
-@fragments.metaData(metaData)
+@fragments.metaData(page)
 @head
 
 @* stylesheet <link>s - get the stylesheets downloading ASAP *@
 @fragments.stylesheets(projectName)
```

Comment:
I'm not sure how `model.Page` differs from `model.metaData` but, just as a heads up, we've just started passing `metaData.isCrosswords` through to stylesheets (https://github.com/guardian/frontend/pull/11219/files#diff-0983be310fb44a148d07ad3af46e62f8R13), will that still be possible?